src/ds5/ds5-device.cpp

```diff
@@ -502,6 +558,7 @@ namespace librealsense
         register_info(RS2_CAMERA_INFO_DEBUG_OP_CODE, std::to_string(static_cast<int>(fw_cmd::GLD)));
         register_info(RS2_CAMERA_INFO_ADVANCED_MODE, ((advanced_mode) ? "YES" : "NO"));
         register_info(RS2_CAMERA_INFO_PRODUCT_ID, pid_hex_str);
+        register_info(RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR, usb_type_str);
```

Comment:
I think that the registration should also be conditional
Alternatively, the usb_type_str should be defaulted to 3 (3.1?)